scripts/ts-bin/pair_comments_for_relativerating.ts

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/*
+Usage: Convert answers with multiple parts into separate rows.
+
+  ts-node ./ts-bin/pair_comments_for_relativerating.ts \
+    --infile="./tmp/path-to-input.jsonl" \
+    --output_path_prefix="./tmp/path-to-paired"
+*/
+
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import * as readline from 'readline';
+// import * as jsonexport from "jsonexport/dist"
+// import * as stream from 'stream';
+
+// Command line arguments.
+interface Params {
```

Comment:
More style nitpicking about spaces after ":" characters. =) Here and elsewhere. I think running a linter should enforce this, as well as various other style things throughout the file (let/const, etc).

Reply:
Ran auto-formatter... not very convinced about these kind of space things, but it's now official layout. Nice to have robots do it for you. :)